### PR TITLE
files for react week1 homework is added

### PR DESCRIPTION
**Pull request by DlnyaMazhari**

### CODE REVIEW

File: react/week1/todo-app/src/activities.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  );
+}
+
+function Activities(props) {
+  return (
+    <>
+      <HeaderTitle />
+
+      {props.TodoList.map((todo) => {
+        return <ListItems task={todo.task} deadlineDate={todo.deadlineDate} />;
```
Comment (keerthi1822) [inline react/week1/todo-app/src/activities.js:23]: Seems like you are creating different orderlist for every item here. Can you try to modify the code that all the list items are under single ol.Thanks

Comment (DlnyaMazhari) [reply react/week1/todo-app/src/activities.js:23]: thanks for the feedback but Its one header with three items , its not different order list for each item ! I have the capture but couldnt post it here .

File: react/week1/todo-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  {
+    task: "Eat breakfast",
+    deadlineDate: "Fri Sep 15 2017",
+  },
+];
+
+function App() {
+  return (
+    <div>
```
Comment (khjcph) [inline react/week1/todo-app/src/App.js]: you actually don't need a div here

File: react/week1/todo-app/src/activities.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+function HeaderTitle() {
+  return (
+    <>
+      <h1>Todo list</h1>
+    </>
+  );
+}
+
+function ListItems(props) {
```
Comment (khjcph) [inline react/week1/todo-app/src/activities.js:7]: maybe this should just be `ListItem`?

When passing props, you can actually use a very cool javascript feature called destructuring. For example

    const ListItems = ({description, deadline}) => { }

Then you can access the variables without using `props.`

File: react/week1/todo-app/src/activities.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+function HeaderTitle() {
+  return (
+    <>
```
Comment (khjcph) [inline react/week1/todo-app/src/activities.js]: you don't actually need the fragments here

File: react/week1/todo-app/src/activities.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      <h1>Todo list</h1>
+    </>
+  );
+}
+
+function ListItems(props) {
+  return (
+    <>
```
Comment (khjcph) [inline react/week1/todo-app/src/activities.js]: also here you don't need the fragment

**Review by khjcph — APPROVED** [review]
Great work!! 👍

I left a few comments that you can look at if you want. I don't remember if we covered destructuring in class. If not, you can just ignore that.

File: react/week1/todo-app/src/App.js (hunk 2 above)
Comment (DlnyaMazhari) [reply react/week1/todo-app/src/App.js]: I am not sure why but ok , I removed !

File: react/week1/todo-app/src/activities.js (hunk 4 above)
Comment (DlnyaMazhari) [reply react/week1/todo-app/src/activities.js]: done!

File: react/week1/todo-app/src/activities.js (hunk 5 above)
Comment (DlnyaMazhari) [reply react/week1/todo-app/src/activities.js]: done!

File: react/week1/todo-app/src/activities.js (hunk 3 above)
Comment (DlnyaMazhari) [reply react/week1/todo-app/src/activities.js:7]: ok , thank you .

**Pull request merged**